GetIntoTeachingApi/Models/SchoolsExperience/SchoolsExperienceSignUp.cs

```diff
@@ -132,5 +150,25 @@ private void AcceptPrivacyPolicy(Candidate candidate)
                 };
             }
         }
+        
+        private void AddQualification(Candidate candidate)
+        {
+            if (ContainsQualification() && !candidate.Qualifications.Any())
+            {
+                candidate.Qualifications.Add(new CandidateQualification()
+                {
+                    Id = QualificationId,
+                    UkDegreeGradeId = UkDegreeGradeId,
+                    DegreeStatusId = DegreeStatusId,
+                    DegreeSubject = DegreeSubject,
+                    TypeId = DegreeTypeId ?? (int)CandidateQualification.DegreeType.Degree
+                });
+            }
+        }
+        
+        private bool ContainsQualification()
+        {
+            return UkDegreeGradeId != null || DegreeStatusId != null || DegreeSubject != null || DegreeTypeId != null;
```

Comment:
Are any of these fields mandatory for a Degree to be valid ? Vould missing fiedls here cause problems with the downstream systems ?

Reply:
nope, they are intentionally optional as older clients will not send the new degree fields. Missing these values means that a qualification record will not be created in the CRM (as is currently the case), but shouldn't break anything downstream.